samples/Config.CustomConfigurationProviders.Sample/Microsoft.Extensions.Configuration.ConfigFile/ConfigFileConfigurationProvider.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+
+namespace Microsoft.Extensions.Configuration.ConfigFile
+{
+    /// <summary>
+    /// This ConfigurationProvider tries to parse .config files from System.Configuration
+    /// https://msdn.microsoft.com/en-us/library/system.configuration.configuration(v=vs.110).aspx
+    /// It can process configurations in the form: 
+    ///     1. &lt;add key=&quot;&quot; value=&quot;&quot; /&gt;
```

Comment:
Thanks for writing doc comments!  Though, these do not render properly in the IDE. Give it a shot by hovering over an instance with your mouse pointer.  For example, lists like above need to be formatted as described here: https://msdn.microsoft.com/en-us/library/y3ww3c7e.aspx